mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -45,23 +37,33 @@ String dockerImage() {
 
 pipeline {
     agent none
+    parameters {
+        booleanParam(name: 'sharedLib', defaultValue: true,
+            description: 'Run shared library tests')
+        booleanParam(name: 'staticLib', defaultValue: true,
+            description: 'Run static library and MIOpen integration tests')
+    }
     stages {
         stage('Build and Test') {
             parallel {
                 stage("Shared Library: fixed E2E tests") {
+                    when {
+                        beforeAgent true;
+                        equals expected: true, actual: params.sharedLib;
+                    }
                     agent {
                         docker {
                             image dockerImage()
                             args dockerArgs()
-                            label "rocm"
+                            label "gfx906"
```

Comment:
I think only static library does tuning and needs to run on gfx906 only. We can still use gfx900 on shared library tests.

Reply:
Ah, I didn't know `rocm` implied a GPU. I'll roll this change back